addons/main.go

```diff
@@ -170,7 +182,10 @@ func main() {
 		Port:                   flags.webhookServerPort,
 		CertDir:                constants.WebhookCertDir,
 		LeaderElection:         flags.enableLeaderElection,
-		LeaderElectionID:       "5832a104.run.tanzu.addons",
+		LeaderElectionID:       flags.leaderElectionID,
+		LeaseDuration:          &flags.leaderElectionLeaseDuration,
+		RenewDeadline:          &flags.leaderElectionRenewDeadline,
+		RetryPeriod:            &flags.leaderElectionRetryPeriod,
```

Comment:
Where do these parameters get used?

Reply:
Hi Marjan, thanks for the review. These parameters are in manager `Options` and will be used by `client-go` library, which will take care of the actual leader election logic.

Reply:
Thanks Leo for the clarification, I see these are defined here: https://kubernetes.io/docs/reference/command-line-tools-reference/kube-controller-manager/.